piggy_bankers_app/View/SignUpView.swift

```diff
@@ -8,7 +8,11 @@
 import SwiftUI
 
 struct SignUpView: View {
-    @EnvironmentObject var userManager: UserManager
+    @State var firstName: String = ""
+    @State var lastName: String = ""
+    @State var username: String = ""
+    @State var password: String = ""
+    @State var confirmPassword: String = ""
```

Comment:
Does it make sense to keep these as state vars, and then when the user submits the from we bundle them and pass them to the VM to make a new user? That's what makes sense to me

Reply:
Should have left them up. Wanted to give you the freedom of handling auth how you wanted.